oidc-ui/configure_start.sh

```
@@ -0,0 +1,88 @@
+
+    echo "unzip image files.."
+    chmod 775 $image_path/*
+    cd $image_path
+    unzip -o esignet-image.zip
+    rm esignet-image.zip
+    echo "unzip image completed."
+fi
+
+#sign-in-button-plugin
+echo "unzip plugins.."
+cd $plugins_path/temp
```

Shellcheck (0.11.0)

[warning] plugins_path is referenced but not assigned.

(SC2154)